### PR TITLE
AutoDoc + small tweaks

### PR DESCRIPTION
**Pull request by cesare-montresor**
AutoDoc.cs
- Added support classes for Misc

Misc.cs
-  added autodoc comments

Mobile.cs
- Fix typo of property. small example of "deprecation via AutoDoc", the old version continues to work, but only the new one shows in the docs.

Player.cs
- Added comments about the double party feature

PythonEngine.cs
- Fix wrong path for libraries in case of CUO ( good old wrong relative path )

### CODE REVIEW

File: Razor/RazorEnhanced/PythonEngine.cs [hunk 1]
```diff
@@ -30,15 +30,15 @@ class PythonEngine
             // When run via CUO the paths are messed up, so we ditch the default ones and put the correct ones
             paths.Add(basepath);
             paths.Add(Misc.CurrentScriptDirectory());
-            paths.Add(Path.Combine(basepath, "Lib"));
+            paths.Add(Path.Combine(basepath, "Libs"));
```
Comment (credzba) [inline Razor/RazorEnhanced/PythonEngine.cs]: MUST be Lib, there is no S on the end of the directory

**Review by credzba — CHANGES_REQUESTED** [review]
Path for Pytghon 3.4 libraries is named Lib not Libs

File: Razor/RazorEnhanced/PythonEngine.cs (hunk 1 above)
Comment (cesare-montresor) [reply Razor/RazorEnhanced/PythonEngine.cs]: path fixed

**Comment by cesare-montresor** [comment]
fixed path

**Review by credzba — APPROVED** [review]

**Pull request merged**